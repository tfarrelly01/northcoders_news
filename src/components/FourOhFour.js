import React from 'react';

const FourOhFour = () => (
  <div className="modal is-active">
    <div className="modal-background" />
    <div className="modal-content">
      <section className="box">
        <div className="column is-12">
          <h1 className="title">
            <b>Oh No 404!</b>
          </h1>
          <h2>Unfortunately the page that you requested cannot be found</h2>
        </div>
      </section>
    </div>
  </div>
);

export default FourOhFour;
