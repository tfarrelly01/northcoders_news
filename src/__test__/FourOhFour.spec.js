import React from 'react';
import { shallow } from 'enzyme';
import FourOhFour from '../components/FourOhFour';

test('FourOhFour renders correctly', () => {
  const component = shallow(<FourOhFour />);
  expect(component).toMatchSnapshot();
});