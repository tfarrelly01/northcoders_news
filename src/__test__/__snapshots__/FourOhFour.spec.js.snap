// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="modal is-active"
  >
    <div
      className="modal-background"
    />
    <div
      className="modal-content"
    >
      <section
        className="box"
      >
        <div
          className="column is-12"
        >
          <h1
            className="title"
          >
            <b>
              Oh No 404!
            </b>
          </h1>
          <h2>
            Unfortunately the page that you requested cannot be found
          </h2>
        </div>
      </section>
    </div>
  </div>,
  "nodes": Array [
    <div
      className="modal is-active"
    >
      <div
        className="modal-background"
      />
      <div
        className="modal-content"
      >
        <section
          className="box"
        >
          <div
            className="column is-12"
          >
            <h1
              className="title"
            >
              <b>
                Oh No 404!
              </b>
            </h1>
            <h2>
              Unfortunately the page that you requested cannot be found
            </h2>
          </div>
        </section>
      </div>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <FourOhFour />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="modal is-active"
        >
          <div
            className="modal-background"
          />
          <div
            className="modal-content"
          >
            <section
              className="box"
            >
              <div
                className="column is-12"
              >
                <h1
                  className="title"
                >
                  <b>
                    Oh No 404!
                  </b>
                </h1>
                <h2>
                  Unfortunately the page that you requested cannot be found
                </h2>
              </div>
            </section>
          </div>
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="modal is-active"
        >
          <div
            className="modal-background"
          />
          <div
            className="modal-content"
          >
            <section
              className="box"
            >
              <div
                className="column is-12"
              >
                <h1
                  className="title"
                >
                  <b>
                    Oh No 404!
                  </b>
                </h1>
                <h2>
                  Unfortunately the page that you requested cannot be found
                </h2>
              </div>
            </section>
          </div>
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <FourOhFour />,
}
`;
